### ISSUE TITLE
Ex Falso - Slowdown Renaming Many Files

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#36]
Original [issue 36](https://code.google.com/p/quodlibet/issues/detail?id=36) created by xyhthyx on 2008-10-19T18:57:36.000Z:

When you rename many files at once in Ex Falso, the gtk.TreeView updates
after every single rename, causing the renaming process to choke. 

If you rename 1-20 files the performance is ok, but if you select 120 songs
like I did and select them all to rename them to fix an incorrect artist
name for example, the performance is poor.

I've attached a simple patch for the file quodlibet/qltk/renamefiles.py
which fixes this.

Basically I added a self.view.freeze_child_notify() and
self.view.thaw_child_notify() before and after the rename process begins to
stop the tree from updating until the renaming is complete.

I tested the patch with a 95 song mass rename and it performed very well.

The diff is against the quodlibet-2.0.tar.gz release.
